src/wallet/wallet2.cpp

```diff
@@ -493,22 +496,31 @@ size_t estimate_rct_tx_size(int n_inputs, int mixin, int n_outputs, size_t extra
   // ecdhInfo
   size += 2 * 32 * n_outputs;
   // outPk - only commitment is saved
-  size += 32 * n_outputs;
+  if (!bulletproof)
+    size += 32 * n_outputs;
   // txnFee
   size += 4;
 
-  LOG_PRINT_L2("estimated rct tx size for " << n_inputs << " with ring size " << (mixin+1) << " and " << n_outputs << ": " << size << " (" << ((32 * n_inputs/*+1*/) + 2 * 32 * (mixin+1) * n_inputs + 32 * n_outputs) << " saved)");
+  LOG_PRINT_L2("estimated rct tx size for " << n_inputs << " with ring size " << (mixin+1) << " and " << n_outputs << ": " << size << " (" << ((32 * n_inputs/*+1*/) + 2 * 32 * (mixin+1) * n_inputs + (bulletproof ? 0 : (32 * n_outputs))) << " saved)");
```

Comment:
I'm not sure what this change means. Could you elaborate?

Reply:
Moot now since I swapped V and outPk saving. I'll remove.